### PR TITLE
PSMDB-1057 Release notes 4.2.20-20

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/conf.py
	new file:   source/release_notes/4.2.20-20.rst
	modified:   source/release_notes/index.rst

### CODE REVIEW

File: source/release_notes/4.2.20-20.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+* `SERVER-59435 <https://jira.mongodb.org/browse/SERVER-59435>`_ - Fixed the double free state for the ``DocumentSource::optimizeAt()`` pipeline by making sure all pointers are in place before disposing the pipeline prefix     
+
+  
+ Find the full list of changes in the `MongoDB 4.2.20 Community Edition release notes <https://www.mongodb.com/docs/manual/release-notes/4.2/#4.2.20---may-9--2022>`_
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-1011`: Add KMIP master key rotation
+* :jirabug:`PSMDB-1011`: The ``kmipClientCertificateFile`` option now includes both the client private key and public certificate
```
Comment (igorsol) [inline source/release_notes/4.2.20-20.rst]: This line should refer to PSMDB-1043

**Review by igorsol — CHANGES_REQUESTED** [review]
Is the `4.4.14.rst` file added to this PR by accident?

**Review by igorsol — APPROVED** [review]
Thanks

File: source/release_notes/4.2.20-20.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.2/install/index.html>`_
+
+Percona Server for MongoDB 4.2.20-20 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 4.2.20 Community Edition.
+It supports MongoDB 4.2.20 protocols and drivers.
+
+Release Highlights
+==================
+
+|PSMDB| now supports the master key rotation for data encrypted using the  :ref:`Keys Management Interoperability Protocol (KMIP) <kmip>` protocol (tech preview feature). This improvement allows users to comply with regulatory standards for data security.
```
Comment (spron-in) [inline source/release_notes/4.2.20-20.rst:17]:
```suggestion
|PSMDB| now supports the master key rotation for data-at-rest encryption using the  :ref:`Keys Management Interoperability Protocol (KMIP) <kmip>` protocol (tech preview feature). This improvement allows users to comply with regulatory standards for data security.
```

**Review by spron-in — APPROVED** [review]

**Pull request merged**